src/coremods/core_lusers.cpp

```diff
@@ -90,7 +90,7 @@ CmdResult CommandLusers::Handle(User* user, const Params& parameters)
 
 	counters.UpdateMaxUsers();
 
-	user->WriteNumeric(RPL_LUSERCLIENT, InspIRCd::Format("There are %d users and %d invisible on %d servers",
+	user->WriteNumeric(RPL_LUSERCLIENT, InspIRCd::Format("There are %lu users and %d invisible on %zu servers",
```

Comment:
`%zu` is not available in the v3 branch as it targets C++98 w/ TR1. This should go into master instead.

Reply:
i think this change is required once we change the type of the parameter, so maybe the whole thing should be rebased on master? might be better given the scope of the change too. wdyt?

Reply:
It's probably fine to just cast it to unsigned long for v3 and change it for master. 4,294,967.295 is more users than any IRC server has ever seen and I doubt that'll be changing soon.

You can rebase against master if you want though. It'll probably be safer to do that.

Reply:
i'll do that. i'll close this and open a new one as trying to fix everything up to base on master is likely going to be hard.